packages/backend/src/routers/apiV1Router.ts

```diff
@@ -63,9 +65,35 @@ apiV1Router.post('/login', passport.authenticate('local'), (req, res, next) => {
 apiV1Router.get(
     lightdashConfig.auth.okta.loginPath,
     storeOIDCRedirect,
-    passport.authenticate('okta', {
-        scope: ['openid', 'profile', 'email'],
-    }),
+    async (req, res) => {
```

Comment:
let's move this handler to `authentication.ts` it could be called `initiateOktaOpenIDLogin`

Reply:
done!